### PR TITLE
Bump Node.js from 10.23.1 to 10.24.1

### PR DESCRIPTION
**Pull request by tmarkley**
### Description
Bumps [node](https://github.com/nodejs/node) from 10.23.1 to 10.24.1
- [Release notes](https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V10.md#10.24.1)

### Testing

Some basic manual smoke tests were run locally as well as running all tests in Jenkins:

Screen Shot 2021-06-24 at 3 18 45 PM

 
### Issues Resolved
Last Node.js bump before tackling #406 
 
### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by ananzh — APPROVED** [review]
LGTM

**Review by kavilla — COMMENTED** [review]
Will these commits be squashed, I see the chromium commit without a description and probably would want a reason for updating the chromium version.

**Comment by tmarkley** [comment]
Updated the commit message for the `chromedriver` change.

**Review by ananzh — APPROVED** [review]
Nice. I tested these two commits yesterday and it fixed my jenkins pipeline

**Review by kavilla — DISMISSED** [review]
Thanks for the changes!

**Comment by tmarkley** [comment]
We may need to get an admin to "remove the outdated check in the protected branch settings of the repository settings" as referenced in this GitHub community post: https://github.community/t/github-actions-pull-requests-always-include-some-outdated-checks/16157

**Review by ananzh — APPROVED** [review]
nice. thanks tommy

File: .github/workflows/pr_check_workflow.yml [hunk 1]
```diff
@@ -14,16 +14,12 @@ jobs:
...
 
-    strategy:
-      matrix:
-        node-version: [10.23.1]
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
```
Comment (kavilla) [inline .github/workflows/pr_check_workflow.yml:25]: This was the magic sauce to make it build? Nice.

**Review by kavilla — APPROVED** [review]

File: .github/workflows/pr_check_workflow.yml (hunk 1 above)
Comment (tmarkley) [reply .github/workflows/pr_check_workflow.yml:25]: It was building without this, but I wanted to simplify. Getting all of the checks to pass still required admin intervention, and I documented the steps in [this comment](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/544#issuecomment-871094121).

**Pull request merged**